pkgs/applications/misc/hydra/default.nix

```diff
@@ -0,0 +1,33 @@
+{ fetchFromGitHub, buildGoModule, lib, stdenv }:
+
+buildGoModule rec {
+  pname = "hydra";
+  version = "1.10.2";
+
+  src = fetchFromGitHub {
+    owner = "ory";
+    repo = "hydra";
+    rev = "v${version}";
+    sha256 = "01q4ihhf06iyk84ssnz9r479zg5n9bnj58pv8fcfjm9ai1mvb1il";
+  };
+
+  vendorSha256 = "1ba2g2fzhpml5b8kz2y15ng83l7884mzn6rl1qj4dlwz6vhhx6qq";
+
+  subPackages = [ "." ];
+
+  buildFlags = [ "-tags sqlite" ];
+
+  doCheck = false;
```

Comment:
Why did you disable tests?

Reply:
I copied from Kratos.

Reply:
Does it not have tests? Do they not work? We can't just disable tests.

Reply:
@CMCDragonkai Please address @SuperSandro2000's review request - check if the tests don't fail, and if they do and you can't make them succeed, write a comment above `doCheck = false;` explaining what's wrong.

Reply:
Apologies my initial need for this package has been de-prioritised atm.

Reply:
> Apologies my initial need for this package has been de-prioritised atm.

It's OK. Let's not keep this open and stale. I'll close the PR and we can continue in the future. Feel free to request my review.